src/content/docs/plugins/submission.md

```diff
@@ -1,4 +1,115 @@
 ---
-title: Plugin Submission
-description: How to submit your plugin to the official Vencord repository
+title: Plugin Submission & Updates
+description: Vencord Plugin submission and updates guide
 ---
+
+This guide describes steps on how to release a new plugin to Vencord and to make changes to the already existing plugin. 
+
+## Initial Plugin Submission 
+
+### Pre-submission Checklist 
+
+- The folder in which stores your plugins should have camelCase name (for instance, `myAwesomePlugin`). 
+- `README.md` is in your plugin folder with a description, usage instructions, screenshots(if any) 
```

Comment:
Suggest users to attach images in GitHub issue comments. There's an issue thread for it

Reply:
Mentioned it 👍🏻